src/tokens/functional/color/dark/patterns-dark.json5

```diff
@@ -1531,6 +1531,36 @@
         },
       },
     },
+    inactive: {
```

Comment:
Will we need this design for anything other than a button? Like an action list item or toggle switch, etc etc.

In CSS, would you just use the `rest` token for hover/active state?

Reply:
> Will we need this design for anything other than a button? Like an action list item or toggle switch, etc etc.

Yes, we will also have "inactive" states for ActionList items. We haven't identified any other controls that need to support this state.

Here's the ActionList.Item @primer/react PR: https://github.com/primer/react/pull/3913

I was able to just use `fgColor.muted`, but we should probably add component tokens. What do you think?

---

> In CSS, would you just use the rest token for hover/active state?

Yup, inactive buttons won't have hover and active states.

Reply:
Just thinking we might want to move these tokens you added into the `control` pattern instead of `button` if we're able to reuse them elsewhere. But if they need to be custom for each type of control, then leaving them here in `button` makes sense.

Reply:
I think they'll need to be different for each type of control.